app/sqlitehelper.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
+from sqlite3 import Connection, Error
+
+DEFAULT_DATABASE = "sqlitedatabase.db"
+
+class SQLiteHelper:
+    """Class to manage data stored in an SQLite database"""
+    def __init__(self, db=DEFAULT_DATABASE) -> None:
+        self.db = db
+        self.connection = self.create_connection()
+
+    def create_connection(self) -> Connection:
+        """Open and return Connection to the SQLite database."""
+        connection = None
+
+        try:
+            connection = sqlite3.connect(self.db)
+        except Error as e:
+            print("Error connecting to database: ", e)
+
+        return connection
+
+    def close_connection(self):
+        """Close the Connection to the database"""
+        try:
+            self.connection.close()
+        except Error as e:
+            print("Error closing connection to database: ", e)
+
+    def get_all_data(self):
+        print("I'm talking to the database right now....connection: ", self.connection)
```

Comment:
What does printing self.connection show?

Reply:
It just prints the sqlite3 Connection object. I initially had that to debug since the connection could'be been null. After the latest commit, that is not possible and I have removed this since it wasn't very relevant or helpful anymore.